frontend/src/metabase/models/components/ModelDetailPage/ModelActionDetails/ModelActionListItem.tsx

```diff
@@ -42,11 +44,21 @@ function ModelActionListItem({ action, onEdit }: Props) {
       <ImplicitActionCardContent />
     ) : null;
 
+  const subtitleParts = [
+    action.public_uuid && t`Public Action`,
+    // Remove this optional chaining after removed all the existing actions without creators
```

Comment:
Not sure if we could justify leaving this comment, but I want to stress that the optional chaining below **should** be removed eventually. It'd be easier to do that when we could archive actions though.